akka-docs/rst/scala/persistence.rst

```diff
@@ -655,16 +655,16 @@ adaptation simply return ``EventSeq.empty``. The adapted events are then deliver
 
 Persistent FSM
 ==============
-``PersistentFSMActor`` handles the incoming messages in an FSM like fashion.
+``PersistentFSM`` handles the incoming messages in an FSM like fashion.
```

Comment:
cc @leonidb this makes the classes more consistent with `FSM` (and how we refer to them anyway "persistent fsm" :smile:)